x/src/lint.rs

```diff
@@ -0,0 +1,164 @@
+use crate::{config::Config, utils::project_root};
```

Comment:
already violated the lint in the first diff, no licensing :P

Reply:
just proving my point :)